.github/workflows/build.yml

```diff
@@ -54,8 +52,12 @@ jobs:
         run: echo ${{ secrets.DOCKER_PASSWORD }} | docker login -u ${{ secrets.DOCKER_USERNAME }} --password-stdin
       - name: Start docker container
         run: docker-compose up -d; sleep 15
-      - name: Send coverage
+      - name: Test Node 10
+        if: matrix.node == '10'
```

Comment:
I would more say if not 14 then test. Like that if we want to add node 12 it just works ^^

Reply:
Or you could even add a variable into the matrix to know if you have to do a coverage or a simple test

Reply:
```yaml
strategy:
  matrix:
    node: ['10', '14']
    include:
      - node: 14
         withCoverage: ${{ true }}
```

```
  - name: Test with Node ${{ matrix.node }}
    if: !matrix.withCoverage
```